packages/lesswrong/components/tagging/TagPage.tsx

```diff
@@ -149,7 +149,7 @@ export const styles = (theme: ThemeType): JssStyles => ({
   },
 });
 
-export const tagPostTerms = (tag: TagBasicInfo | null, query: any) => {
+export const tagPostTerms = (tag: {_id: string, name: string} | null, query: any) => {
```

Comment:
Eh - super minor, but in general I think `Pick` shows the intention better.

```suggestion
export const tagPostTerms = (tag: Pick<DbTag, "_id" | "name"> | null, query: any) => {
```